docs/install/install-cave.md

```diff
@@ -68,17 +68,35 @@ Additionally users can choose to run a [virtual machine (VM)](#method-1-linux-vi
 
 **Latest Version: 18.2.1-6**
 
-For Windows, Unidata offers two installation options: a [**Linux Virtual Machine**](#method-1-linux-virtual-machine), or a [**Direct Windows Installation**](#method-2-direct-windows-install).
+For Windows, Unidata offers two installation options: a [**Direct Windows Installation**](#method-1-direct-windows-install), or a [**Linux Virtual Machine**](#method-2-linux-virtual-machine).
+
+!!! note "As of 4/24/2023, the direct install method has been completely simplified! No admin priviledges or extra software needed."
 
-Currently, the [virtual machine (VM)](#method-1-linux-virtual-machine) is the recommended form of install for those who do not have administrative priviledges on the machine, or beginners who want a simpler installation process.  
 !!! warning "At the moment, the VM option may not render all products in CAVE (ex. RGB composites of satellite imagery)"
 
-The [direct installation method](#method-2-direct-windows-install) is recommended for those who have administrative priviledges and a little bit of experience installing more software.
+### Method 1: Direct Windows Install
+
+This method has been simplified to include python packaged with CAVE so no other software installation is necessary. The CAVE shortcut on the Desktop is a `.bat` file that updates your env variables only when CAVE is launched, so it will not mess with any other python applications or environments.
+
+#### System Requirements
+
+- Running Windows OS
 
+#### Download and Installation Instructions
 
-### Method 1: Linux Virtual Machine
+1. Download and install: [**awips-cave.msi** <i class="fa fa-download"></i>](https://downloads.unidata.ucar.edu/awips2/current/windows/awips-cave.msi)
+    - This will install CAVE to the user level in `%HOMEPATH%\AppDir\Roaming\UCAR Unidata\AWIPS CAVE`.
 
-This method is recommended for beginners, or those with less computer knowledge as it is a very simple installation, however at this time, some CAVE functionality may be missing (ex: rendering RGB satellite images).
+#### Run CAVE
+
+To run CAVE, either:
+
+- Double click on the `CAVE` shortcut on the desktop
+- Type "cave" in the start bar and hit enter
```

Comment:
Does this second method still work?  It pulls up the .bat file by searching?

Reply:
Yep.